lib/party_foul.rb

```diff
@@ -78,9 +78,9 @@ def self.repo_url
   # Will also setup for GitHub api connections
   #
   # @param [Block]
-  def self.configure(&block)
+  def self.configure
     yield self
-    self.github ||= Octokit::Client.new access_token: oauth_token, api_endpoint: api_endpoint
```

Comment:
The `configure` block is most likely called once when the environment is loaded (typically in an initializer in Rails apps), right? If so, I don't see a problem. 

Before this commit, it prevents a call to `configure` from updating the client if `oauth_token` or `api_endpoint` are changed.